### ISSUE TITLE
Way to allow updating without code signing?

### ISSUE THREAD

**Issue by jshamg** [electron/update-electron-app#58]
Hi,

I'm a young dev and just created an open source electron app. I also want to have the feature to let users know that there is an update for my app. The thing is I dont want spend all these fees just to sign my code. There should be a solution where the user gets informed of a new version of an app and worst case is, that he has to manually download the new version... But else this is a pretty useless solution for a lot of small devs...

greetings!

**Comment by kasp1** [comment]
Agreed. If MacOS checks changes to the app, it's understandable. But if Windows doesn't (?), at least Windows ports should be allowed to update without code signing considering how pricey the certs are. If possible.

**Comment by mswgen** [comment]
I agree.
